modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1911,6 +1911,59 @@ definition {
 		}
 	}
 
+	@description = "LPS-179803 - Verify the user cannot delete the only custom field in an object definition"
+	@priority = 5
+	test CannotDeleteSoleCustomField {
+		property custom.properties = "feature.flag.LPS-179803=true";
+
```

Comment:
You're still missing the property to add this to acceptance. Please see other priority 5 tests, like `CannotEditERCField` which include this property